apps/sensenet/src/components/drawer/PermanentDrawer.tsx

```diff
@@ -99,7 +114,11 @@ const PermanentDrawer: React.FunctionComponent<RouteComponentProps> = props => {
           </div>
 
           {matchPath(props.location.pathname, `/:repositoryId/saved-queries`) === null ? (
-            <AddButton isOpened={opened} path={currentPath} />
+            matchPath(props.location.pathname, { path: `/`, exact: true }) === null &&
```

Comment:
I would like to see this refactored. Lets talk about it sometime.

Reply:
I changed the blacklist to a whitelist, so we don't need to create loop for the checks (cause it contains only 3 conditions)